### PR TITLE
Refresh nav on content type creation

### PR DESCRIPTION
**Pull request by itkg-ngilain**

### CODE REVIEW

File: BackofficeBundle/Resources/public/coffee/FullPageFormView.coffee [hunk 1]
```diff
@@ -36,6 +36,7 @@ FullPageFormView = OrchestraView.extend(
     $("form", @$el).ajaxSubmit
       context: button: $(".submit_form",event.target).parent()
       success: (response) ->
+        displayMenu() if $(event.target).attr('name') is 'content_type'
```
Comment (itkg-nanne) [inline BackofficeBundle/Resources/public/coffee/FullPageFormView.coffee]: I would prefer to have a dedicated view for content_type which will be loaded by appConfigurator. This view could override FullPageFormView by adding displayMenu() in initialize.

**Comment by itkg-ngilain** [comment]
@itkg-nanne : update done, but I don't like the overload of the initialize function

**Comment by itkg-nanne** [comment]
An other way could be to bind on ready event after render. In widgetChannel, if view.options.submitted and view.options.afterSubmitted then view.options.afterSubmitted(). You keep same override logic but just with an afterSubmitted method. Could be used for folder, node and template. right?

**Comment by itkg-nanne** [comment]
In case of new ContentType, will it open navigation on contentType created?

**Comment by itkg-ngilain** [comment]
@itkg-nanne : update done introducing an onOrchestraViewReady and onViewReady functions

**Comment by itkg-nanne** [comment]
:+1:  after rebase

**Pull request merged**